### PR TITLE
Python support update

### LINKED ISSUE #5: Update supported Python versions
- drop support for Python versions < 3.6
- add support for Python versions 3.7, 3.8 (if missing)
- fix linting errors (if present)

If you are a beginner, and need to help for this issue, especially during Hacktoberfest, please comment here or reach out to me via https://jugmac00.github.io/

### PR DESCRIPTION
**Pull request by sgaist**
This merge request updates the minimal Python version to 3.7 and adds 3.8.

It also adds the tools required to run the tests to list of dependencies.

Fixes #5

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-python: 3.5
+python: 3.7
 env:
-    - TOXENV=py27
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=py35
+    - TOXENV=py37
+    - TOXENV=py38
     - TOXENV=pep8
```
Comment (jugmac00) [inline .travis.yml]: Please add support for Python 3.6.

Also please see https://github.com/morepath/morepath_cerebral_todomvc/commit/c5b413728e05e92423c162534c32e6b39ee1cb0f on how to fix the currently broken CI.

File: setup.py [hunk 2]
```diff
@@ -45,13 +45,16 @@ def get_long_description():
             'coverage',
             'pytest',
             'webtest',
-            'pyscss'
+            'pyscss',
+            'tox',
+            'flake8'
         ],
```
Comment (jugmac00) [inline setup.py:50]: flake8 seems not to be necessary, as it gets added by tox.

File: setup.py [hunk 3]
```diff
@@ -45,13 +45,16 @@ def get_long_description():
...
     ),
     classifiers=[
         'Intended Audience :: Developers',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
         'Topic :: Software Development :: Libraries :: Python Modules',
```
Comment (jugmac00) [inline setup.py:58]: Please add Python 3.6

File: tox.ini [hunk 4]
```diff
@@ -1,21 +1,21 @@
 [tox]
-envlist = py27,py33,py34,py35,pep8,coverage
+envlist = py37,py38,pep8,coverage
```
Comment (jugmac00) [inline tox.ini:3]: Please add python 3.6

**Review by jugmac00 — CHANGES_REQUESTED** [review]
LGTM - with some comments. Please try to apply the suggested changes.

**Comment by jugmac00** [comment]
Hi @sgaist 

currently, CI is broken, see https://travis-ci.org/github/morepath/more.webassets/builds/731987458

The reason seems that Travis nowadays need the "matrix" syntax for configuring several Python interpreters, also see here 
morepath/morepath_cerebral_todomvc@c5b4137 

As a side note .. I just saw you force-pushed some new changes. While this works, there is one problem with this. Above I added some inline annotations on what needs to be done for this pr - as you force pushed a new state, these do not match any more. On the other hand, if you just added another commit, I could easily see if you applied the requested changes. Do not worry about too many commits, as the maintainer always can squash all changes into one commit on merging.

So, the rule of thumb I follow - when I work together with other people, especially via GitHub, I do not force push.

Thanks for contributing!

**Comment by sgaist** [comment]
@jugmac00 I see your point, and I will do it like that. 

I forced push because I am used to provide sets of commits (even if squashed) that are "clean" in the sense that the kind of small fixes you asked were better served by being integrated in the original work.

**Comment by jugmac00** [comment]
> @jugmac00 I see your point, and I will do it like that.
> 
> I forced push because I am used to provide sets of commits (even if squashed) that are "clean" in the sense that the kind of small fixes you asked were better served by being integrated in the original work.

I absolutely noticed that your commits are super clean! :+1:

File: setup.py (hunk 3 above)
Comment (sgaist) [reply setup.py:58]: Done

File: setup.py (hunk 2 above)
Comment (sgaist) [reply setup.py:50]: Done

File: tox.ini (hunk 4 above)
Comment (sgaist) [reply tox.ini:3]: Done

File: .travis.yml (hunk 1 above)
Comment (sgaist) [reply .travis.yml]: Done

**Comment by jugmac00** [comment]
One last thing - could you please add a change log entry? Thank you!

**Review by jugmac00 — APPROVED** [review]

**Comment by jugmac00** [comment]
Thank you!

**Pull request merged**